DOCUMENTATION.md

```diff
@@ -29,6 +30,12 @@ This module implements all models for Resin Python SDK.
 ## Application
 
 This class implements application model for Resin Python SDK.
+
+Due to the API changes, the returned Application object schema changes as well. Here are the format of old and new returned object.
+
+The old returned object: `should_track_latest_release, app_name, application, __metadata, actor, git_repository, version, user, device_type, support_expiry_date, commit, id`.
+
+The new returned object (since Python SDK v1.7.0): `depends_on__application, should_track_latest_release, app_name, __metadata, is_accessible_by_support_until__date, actor, git_repository, version, user, device_type, commit, id`.
```

Comment:
Are these listed in any particular order? Would it make sense to list them alphabetically?